Habitica/src/main/java/com/habitrpg/android/habitica/APIHelper.java

```diff
@@ -129,6 +132,8 @@ public APIHelper(GsonConverterFactory gsonConverter, HostConfig hostConfig) {
         Crashlytics.getInstance().core.setUserName(this.hostConfig.getUser());
         Amplitude.getInstance().setUserId(this.hostConfig.getUser());
 
+        languageCode = Locale.getDefault().getLanguage();
```

Comment:
should this maybe be the language you set habitica to instead? Eventually we also want to have the app itself be able to be in a specified language. So in those cases, the content should also be in that language. So using the configured language and only using the locale as a fallback might make more sense?